### ISSUE TITLE
Migrate over to using `thenewboston-js` repo

### ISSUE THREAD

**Issue by angle943** [thenewboston-developers/Account-Manager#585]
Make the AccountManager rely on `thenewboston-js` repo fully

**Comment by angle943** [comment]
please get permission from me to work on it before you start

**Comment by tomijaga** [comment]
Can I work on this

**Comment by angle943** [comment]
i don't believe the sdk library is ready yet. talk to @zinoadidi about the progress

**Comment by tomijaga** [comment]
This would be useful for this task.
https://docs.google.com/document/d/1FaPq2stWP0lMtHATNIfLA6ryMd0PtUYdcnrvZbUk7FI/edit
Its a documentation of the similar functions and their locations between Account manager and thenewboston-js

**Comment by tomijaga** [comment]
Hey @angle943, is the list of functions in the document above okay for the account manager or is there anything u would like to add to this list.

**Comment by angle943** [comment]
@tomijaga to be honest I currently don't have the bandwidth to look over the document in detail. But we can start going as is. Are you working in tandem with @zinoadidi about these issues?

**Comment by zinoadidi** [comment]
@angle943 I have already a roadmap item for the switch, I will handle the process from my end with @tomijaga.

Also we are working together to make missing features already, once they are complete, I will kick off milestones for the conversion process

**Comment by angle943** [comment]
@zinoadidi hmm perhaps it might be better to sync up offline later, but question for you, how soon do you think you can incorporate the AccountManager with the SDK? The reason why I ask is that we are soon having a switch to beta, which might end up changing a lot of our BE end points and results

**Comment by zinoadidi** [comment]
yes; I have also considered this as well and the proposal basically is we are going to sync up the SDK development with beta updates as they come; I am working to secure one or two contributors who will be dedicated to this activity.

Concerning Account Manager, there is approximately 1 task pending test from the list of missing functionalities we started to work on a couple of days ago. 
That basically means we can already start switching usage by Friday / weekend.

According to our roadmap; we plan to keep the account manager in sync with SDK no matter what :) and we are ensuring this happens by such kinds of tasks https://github.com/thenewboston-developers/thenewboston-js/issues/88

**Comment by angle943** [comment]
sounds good, here is the migration branch: https://github.com/thenewboston-developers/Account-Manager/tree/javascript-sdk-migration

**Comment by zinoadidi** [comment]
@angle943 All changes are done and ready for review

**Comment by TouchstoneTheDev** [comment]
Is this closed or bounty open ?
